pkg/integrations/metrics/prometheus/global.go

```diff
@@ -0,0 +1,85 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type GlobalHatchetMetric string
+
+const (
+	QueueInvocationsCount                GlobalHatchetMetric = "queue_invocations"
```

Comment:
do you think adding a queue depth gauge here would be useful? not sure how prom handles different types of metrics

Reply:
Yeah so I was going to add tenant-specific metrics in the next PR, which this would be.